package com.remindly.tests;

import com.remindly.core.TestBase;
import org.testng.Assert;
import org.testng.annotations.Test;

public class MainScreenTests extends TestBase {

    @Test
    public void appLaunchTest() {
        Assert.assertTrue(app.getMainScreen().isNoRemPresent());
    }
}
